sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -690,6 +690,7 @@ static struct snd_soc_dai_link *sof_card_dai_links_create(struct device *dev,
 			links[id].num_codecs = ARRAY_SIZE(max_98373_components);
 			links[id].init = max98373_spk_codec_init;
 			links[id].ops = &max_98373_ops;
+			links[id].dpcm_capture = 1;
```

Comment:
the change looks good but can we add a comment saying that this is needed for the feedback capture stream?

Reply:
Done